src/nncf/quantization/algorithms/weight_compression/algorithm.py

```diff
@@ -101,6 +93,8 @@ def get_weight_compression_configuration(
     elif group_size is None and mode in NON_INT8_MODES:
         if mode in [CompressWeightsMode.MXFP4, CompressWeightsMode.MXFP8_E4M3]:
             group_size = 32
+        elif mode in [CompressWeightsMode.CODEBOOK, CompressWeightsMode.CB4_F8E4M3]:
+            group_size = -1
```

Comment:
is it possible to override group size for codebook?

Reply:
Yes, group size will be set to -1 here only if the previous value was None. So if it is explicitly provided and not None, then that value will be used.